verification/simulator.go

```diff
@@ -231,9 +231,44 @@ func GetSimulatorTargets() []TestTarget {
 		},
 		{
 			"DefaultSupport",
-			getTestTarget([]string{"AutoInit", "Simulation", "X509", "Csr", "IsCA", "RotateContext", "ExtendTci", "IsSymmetric"}),
+			getTestTarget([]string{"AutoInit", "Simulation", "X509", "Csr", "IsCA", "RotateContext", "IsSymmetric", "ExtendTci"}),
 			AllTestCases,
 		},
+		{
+			"CertifyKey_TcbValidation",
+			getTestTarget([]string{"AutoInit", "Simulation", "X509", "Csr", "IsCA", "RotateContext", "ExtendTci"}),
+			[]TestCase{DiceTcbValidationTestCase, TcbValidationSimulationTestCase},
+		},
+		{
+			"DeriveChild",
+			getTestTarget([]string{"AutoInit", "X509", "IsCA"}),
+			[]TestCase{DeriveChildTestCase},
+		},
+		{
+			"DeriveChild_Simulation",
+			getTestTarget([]string{"AutoInit", "Simulation", "X509", "IsCA"}),
+			[]TestCase{DeriveChildSimulationTestCase},
+		},
+		{
+			"DeriveChild_PrivilegeEscalation",
+			getTestTarget([]string{"AutoInit", "X509", "IsCA"}),
+			[]TestCase{DeriveChildPrivilegeEscalationTestCase},
+		},
+		{
+			"DeriveChild_InputFlags",
+			getTestTarget([]string{"AutoInit", "Simulation", "InternalDice", "InternalInfo"}),
+			[]TestCase{DeriveChildInputFlagsTestCase},
+		},
+		{
+			"DeriveChild_MaxTCIs",
+			getTestTarget([]string{"AutoInit", "Simulation"}),
+			[]TestCase{DeriveChildMaxTCIsTestCase},
+		},
+		{
+			"DeriveChild_ChangeLocality",
+			getTestTarget([]string{"AutoInit", "Simulation"}),
+			[]TestCase{DeriveChildLocalityTestCase},
+		},
```

Comment:
Adding new test targets for testcases that alter DPE tree (MakeDefault/ Retain parent), that use Locality control, Escalate priveleges etc.,